.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+:v: :construction:
+
+/cc @zendesk/vegemite
+
+### Description
+Describe the original problem and the changes made on this PR.
+
+### Tasks
+- [ ] Include comments/inline docs where appropriate
+- [ ] Write tests
+
+### References
+* JIRA: https://zendesk.atlassian.net/browse/AF-XXX
```

Comment:
Does this need to be `AFSI`/one of the other new projects?

Reply:
As discussed in RL, this might work for now.